src/insights/competition.ts

```diff
@@ -3,13 +3,32 @@ import { PageMeta, TimeMeta } from "./common";
 export interface CompetitionItem {
   id: string;
   competitionId: string;
+  subCompetitionId: string;
   market: string;
```

Comment:
According to the [API](https://dev-api-insights.carbon.network/competition/list), the following fields could be `null`:
- `market`
- `enablePNL`
- `minBalance`
- `enableVolume`
- `makerIncentive`

Do make sure to include `null` in the struct for the fields above.
<img width="442" alt="Screenshot 2024-07-10 at 5 21 50 PM" src="https://github.com/Switcheo/carbon-js-sdk/assets/65583522/d3b6b93d-9e4b-462e-a6b4-1472e01c0748">